de-lib/src/main/java/org/iplantc/de/fileViewers/client/views/PathListViewer.java

```diff
@@ -157,10 +157,11 @@ void handleDropStatus(Object o,
         }
 
         boolean hasCorrectData(Object data){
-
-            boolean isCollection = data instanceof Collection<?>;
+           boolean isCollection = data instanceof Collection<?>;
             boolean isEmpty = ((Collection<?>) data).isEmpty();
             boolean hasDiskResources = ((Collection<?>) data).iterator().next() instanceof DiskResource;
+            boolean filteredData = false;
```

Comment:
I might be missing something but I don't think `filteredData` and `iterable` are used anywhere?